terraform/modules/service/worker+nvm/main.tf

```diff
@@ -1,10 +1,9 @@
 module "service" {
-  source = "git::https://github.com/wellcometrust/terraform.git//ecs/prebuilt/scaling+nvm?ref=v17.1.0"
+  source = "git::https://github.com/wellcometrust/terraform.git//ecs/prebuilt/scaling+nvm?ref=2181671"
 
```

Comment:
Should this be a proper version, looks like a snapshot?

Reply:
It should – I need a review on https://github.com/wellcometrust/terraform-modules/pull/153 so I can get a new version.